assets/javascripts/overview.js

```diff
@@ -289,6 +289,7 @@ function showAddCommentsDialog() {
 
 function addComments(form) {
   const text = form.elements.text.value;
+  let wannaRestart = document.getElementById('wannaRestartCheck').checked ? 1 : 0;
```

Comment:
```suggestion
  let restartRequested = document.getElementById('restartRequestedCheck').checked ? 1 : 0;
```

and all other occurences in this PR accordingly including the Perl variables please

Reply:
Done https://github.com/os-autoinst/openQA/commit/2c24af9fbb696e60daaca9c2e4b11ffe7298dbc4